.github/workflows/root-ci-config/build_root.py

```diff
@@ -273,6 +273,14 @@ def build(options, buildtype, shell_log):
 
         if result != 0:
             die(result, "Failed cmake generation step", shell_log)
+    else:
+        # Print CMake cached config
+        result, shell_log = subprocess_with_log(f"""
+            cmake -S '{workdir}/src' -B '{workdir}/build' -N -L
```

Comment:
Will this line run trigger CMake generation? If so, it will slow down the build quite a bit because it touches a lot of files, updating their timestamps and making the build step take longer. If it only prints the cached config, then everything should be fine.

Reply:
> Will this line run trigger CMake generation?

I checked: it does not touch `CMakeLists.txt` and really only prints the cache. (And one *must* still give the source dir, else it issues a warnings.)